app/views/podcasts/show.html.erb

```diff
@@ -42,7 +42,13 @@
         <div class="card-body episode-card h-auto border-start-0">
           <% if @next_scheduled.present? %>
             <div class="episode-card-info">
-              <p class="episode-card-date"><%= @next_scheduled.released_at.to_date.strftime("%m/%d") %></p>
+              <p class="episode-card-date">
+                <% if @next_scheduled.published_at.present? %>
```

Comment:
I think we should just make this a method in Episode, because we'll need it many places.  `def published_or_released_at` or something?